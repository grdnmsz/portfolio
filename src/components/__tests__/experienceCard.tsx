import React from 'react'
import renderer from 'react-test-renderer'

import { ExperienceCard } from '../'

describe('ExperienceCard', () => {
  const props = {
    frontmatter: {
      title: 'a title',
      dates: '10-10-2020',
      location: 'some place',
      description: 'some description to fill the gaps'
    },
  }

  it('should render correctly', () => {
    const tree = renderer.create(<ExperienceCard {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
