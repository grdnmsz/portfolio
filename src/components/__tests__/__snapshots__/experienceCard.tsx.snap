// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExperienceCard should render correctly 1`] = `
<div
  className="bg-headerColor rounded"
>
  <div
    className="mb-1 px-6 py-6 text-center"
  >
    <div
      className="mb-4 font-bold hover:text-teal-600 text-2xl"
    >
      <a
        className="flex justify-center w-full"
        href="/about"
      >
        a title
      </a>
    </div>
    <span
      className="text-sm"
    >
      10-10-2020
      <br />
      some place
      <br />
      <br />
    </span>
    <div
      className="md:px-52 lg:px-0 sm:px-32 text-sm"
    >
      some description to fill the gaps
    </div>
  </div>
</div>
`;
